hack/binaries.sh

```diff
@@ -13,7 +13,7 @@ export TERM="${TERM:-dumb}"
 
 main() {
   local kubectl_version=v1.21.1
-  local kind_version=v0.11.1
+  local kind_version=v0.16.0
```

Comment:
Note: this update is not required by this PR, but I don't want to revert it now after I added it.

Reply:
I will delete it if it bother anybody too much.